openid-4-verifiable-presentations-1_0.md

```diff
@@ -1155,6 +1155,8 @@ To obtain Verifier's public key for the input to the key agreement to encrypt th
 
 To sign the Authorization Response, the Wallet MUST use a private key that corresponds to a public key made available in its metadata.
 
+Note: For encryption, implementers have a variety of options available through JOSE, including the use of Hybrid Public Key Encryption (HPKE) as detailed in [@I-D.ietf-jose-hpke-encrypt]. 
```

Comment:
don't we need to say which of the two modes we are recommending to use?

> This specification describes two modes of use for HPKE in JWE:
>
> - HPKE JWE Integrated Encryption, where HPKE is used to encrypt the plaintext.
>
> - HPKE JWE Key Encryption, where HPKE is used to encrypt a content encryption key (CEK) and the CEK is subsequently used to encrypt the plaintext.

Reply:
Not sure if we should or need to make that recommendation here - the choice which mode to use should be imho done by an ecosystem / a profile. From an implementers perspective, it is discoverable which mode was used via the JWE parameters (alg, enc, iv, tag), so this would mainly be about reducing options within openid4vp?